tfjs-core/src/environment.ts

```diff
@@ -192,9 +196,9 @@ function parseValue(flagName: string, value: string): FlagValue {
     return value === 'true';
   } else if (`${+ value}` === value) {
     return +value;
+  } else {
+    return value;
```

Comment:
for string, I'd prefer case sensitive.

Reply:
I agree with Yang. Case sensitive would be better for strings.